src/control/server/storage_scm.go

```diff
@@ -35,7 +37,24 @@ import (
 	"github.com/daos-stack/go-ipmctl/ipmctl"
 )
 
-var (
+//go:generate stringer -type=scmState
+type scmState int
+
+const (
+	unknown scmState = iota
```

Comment:
While it may seem verbose, from a readability perspective it would be better to prefix these state names with the type. e.g. scmStateUnknown, scmStateNoRegions, etc.

That way there is no ambiguity in the reader's mind when they are seeing them used in the codebase.

Reply:
sure, I do it with some of the other types and will do here as well